backend/controller/ingress/ingress.go

```diff
@@ -60,13 +60,12 @@ func matchSegments(pattern, urlPath string, onMatch func(segment, value string))
 }
 
 func ValidateCallBody(body []byte, verb *schema.Verb, sch *schema.Schema) error {
-	var requestMap map[string]any
-	err := json.Unmarshal(body, &requestMap)
+	var root any = nil
+	err := json.Unmarshal(body, &root)
 	if err != nil {
-		return fmt.Errorf("HTTP request body is not valid JSON: %w", err)
```

Comment:
It's odd to me that these checks are in the ingress package even though they check the validity of request data for all requests.
I've removed "HTTP" from the error message because this is being used outside of HTTP use cases

Reply:
It doesn't make much sense for this function to be in this package anymore really.